openedx/core/djangoapps/credit/tasks.py

```diff
@@ -50,6 +42,94 @@ def update_course_requirements(course_id):
         LOGGER.info('Requirements added for course %s', course_id)
 
 
-def get_min_grade_for_credit(course):
-    """ Returns the min_grade for the credit requirements """
-    return getattr(course, "min_grade", 0.8)
+def _get_min_grade_for_credit(course):
+    """ Returns the min_grade for the credit requirements
+
+     Args:
+        course(Course): The course object
+
+    Returns:
+        Float value of minimum_grade_credit attribute of course
+    """
+    return getattr(course, "minimum_grade_credit", 0.8)
```

Comment:
I think we don't need the hard coded default value, just use `getattr(course, "minimum_grade_credit")`. Because this field will be already set on all credit courses.